include/mbedtls/ctr_drbg.h

```diff
@@ -45,7 +48,13 @@
 #define MBEDTLS_ERR_CTR_DRBG_FILE_IO_ERROR                -0x003A  /**< Read or write error in file. */
 
 #define MBEDTLS_CTR_DRBG_BLOCKSIZE          16 /**< The block size used by the cipher. */
+
+#if !defined(MBEDTLS_CTR_DRBG_USE_128_BIT_KEY) /* Use default 256-bit key for CTR-DRBG. */
 #define MBEDTLS_CTR_DRBG_KEYSIZE            32 /**< The key size used by the cipher. */
+#else
+#pragma message("Warning: using smaller (128-bit) key size for CTR DRBG may reduce the security of some operations.")
```

Comment:
Please clarify “may reduce the security of some operations”. I'd say something like “limits the strength of generated material to 128 bits”. See the discussion about the wording of `\warning` above.